src/components/mofo-footer/mofo-footer.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+export default React.createClass({
+  propTypes: {
+    footerLinks: React.PropTypes.arrayOf(React.PropTypes.shape({
+      iconType: React.PropTypes.oneOf([`info`, `email`, `github`, `chat`, `twitter`, `facebook`, `cc-license`, `code-of-conduct`, `cookies`, `legal`, `privacy`, `donate`,]).isRequired,
+      link: React.PropTypes.string.isRequired,
+      text: React.PropTypes.string.isRequired,
+      className: React.PropTypes.string
+    }).isRequired).isRequired,
+    orgs: React.PropTypes.arrayOf(React.PropTypes.shape({
+      name: React.PropTypes.string.isRequired,
+      link: React.PropTypes.string.isRequired,
+      description: React.PropTypes.element.isRequired,
+      className: React.PropTypes.string
+    }).isRequired).isRequired
+  },
+  renderFooterLinks() {
+    return this.props.footerLinks.map((footerLink) => {
+      let className = footerLink.className ? `col-auto ${footerLink.className}` : `col-auto`;
+
+      return (
+        <li className={className} key={footerLink.text}>
+          <a className={`footer-link-${footerLink.iconType}`} href={footerLink.link}>{footerLink.text}</a>
+        </li>
+      );
+    });
+  },
+  renderOrgs() {
+    return this.props.orgs.map((org) => {
+      let orgNameAsClassName = org.name.replace(/\s+/g, `-`).replace(/[^A-Za-z0-9_-]+/g, ``).toLowerCase();
+      let className = `org-info ${orgNameAsClassName}`;
+
+      className = org.className ? `${className} ${org.className}` : className;
```

Comment:
Now I think of it, I think we don't need to generate `orgNameAsClassName` at all. If a custom classname is needed people can just pass it as `org.className`. I will update the code here.